src/components/shared/Forms/Collection/BaseCollectionForm.js

```diff
@@ -182,10 +184,16 @@ class BaseCollectionForm extends Component {
 
   handleSourceChange = (source) => {
     const { errors } = this.state;
-    delete errors.source;
+    delete errors.series;
```

Comment:
What is `series` ?
keep `source`

Reply:
I use same error for tags and source (one of this must be required). remane to sourceOrTag